webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -198,7 +200,7 @@ class ManageManifestModal extends Component {
                           {getManifestName()}
                         </Col>
                       </Row>
-                      {isManifestImported && manifestExpirationDate &&
+                      {isManifestImported && !!manifestExpirationDate &&
```

Comment:
I think a direct check for falsiness, `instanceof Date`, or even `!== null` would be clearer. I just haven't encountered this `!!` syntax much before; what are your thoughts?

Reply:
It's kinda like `Foreman::Cast.to_bool` except not as cool :)

I'm taking advantage here of the fact that `0` is falsy in JavaScript. If `manifestExpirationDate` is `0`, it would evaluate to the 01 Jan 1969 date and I won't want it rendered.

I could just leave it as `manifestExpirationDate` (without the `!!`), but that would risk actually rendering a "0" on the page somewhere. But `false` won't be rendered.

![image](https://github.com/Katello/katello/assets/22042343/b1299aa2-d818-47a1-8f9f-ada7a862d274)

Would you be okay with `Boolean(manifestExpirationDate)`? That's a bit more explicit and I think it will provide the same result.

Reply:
I'm totally ok with the explicit cast; that sounds like a good plan.